cvrf2csaf/common/utils.py

```diff
@@ -13,6 +13,23 @@ def critical_exit(msg, status_code=1):
     exit(status_code)
 
 
+def handle_boolean_config_values(key, val):
+    try:
+        if isinstance(val, bool):
+            return val
+        if val.strip().lower() in {'true', 'yes', '1', 'y'}:
+            return True
+        if val.strip().lower() in {'false', 'no', '0', 'n'}:
+            return False
+
+        critical_exit(f"Reading config.yaml failed. "
```

Comment:
Maybe `raise ValueError` instead and only catch a `ValueError` - that removes the duplication and any unforeseen `Exception` not caught can be handled upstream on caller site.